src/applications/vaos/appointment-list/components/PastAppointmentsList/index.jsx

```diff
@@ -15,7 +15,7 @@ import {
 import ConfirmedAppointmentListItem from '../cards/confirmed/ConfirmedAppointmentListItem';
 import PastAppointmentsDateDropdown from './PastAppointmentsDateDropdown';
 
-export function getPastAppointmentDateRangeOptions(today = moment()) {
+function getPastAppointmentDateRangeOptions(today = moment()) {
```

Comment:
I think this change is breaking other tests.

Reply:
Fixed it, I think this was in an effort to dedupe but it failed greatly.